nextflow-lib/nextflow.config

```diff
@@ -34,9 +34,9 @@ profiles {
     }
     gls {
         params {
-            SARS2_FA = "gs://prj-int-dev-covid19-nf-gls/data/NC_045512.2.fa"
-            SARS2_FA_FAI = "gs://prj-int-dev-covid19-nf-gls/data/NC_045512.2.fa.fai"
-            SECRETS = "gs://prj-int-dev-covid19-nf-gls/data/projects_accounts.csv"
+            SARS2_FA = "gs://prj-int-dev-covid19-nf-gls-poc/data/NC_045512.2.fa" //For POC project
+            SARS2_FA_FAI = "gs://prj-int-dev-covid19-nf-gls-poc/data/NC_045512.2.fa.fai" //For POC project
+            SECRETS = "gs://prj-int-dev-covid19-nf-gls-poc/data/projects_accounts.csv" //For POC project
```

Comment:
Changes here arre unnecessary. NC_045512.2.fa is the SARS-CoV-2 reference genome and NC_045512.2.fa.fai is its index. projects_accounts.csv is a list of IDs and passwords for submission. You do not need to change them.